### PR TITLE
Nc news 7

### PR DESCRIPTION
**Pull request by sunilkrishna49**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -234,20 +248,65 @@ describe("app", () => {
...
+      };
+      return request(app)
+        .post(`/api/articles/${article_id}/comments`)
+        .send(newComment)
+        .expect(404)
+        .then(({ body }) => {
+          expect(body.msg).toBe("Article not found");
+        });
+    });
+  });
```
Comment (B7Bprog) [inline __tests__/app.test.js:344]: Please add another test where extra properties are ignored. This should be a successful post. For example when posting with this:

```
 const newComment = {
        username: "butter_bridge",
        body: "This is a new comment",
        extraProperty: "someValue"
      };
```

Comment (B7Bprog) [inline __tests__/app.test.js:344]: Please write another test where the object is missing required field(s), e.g. no username or body properties and it results in a bad request.

Comment (B7Bprog) [inline __tests__/app.test.js:344]: Please write another test where the given username is not found.

**Review by B7Bprog — CHANGES_REQUESTED** [review]
Hey Sunil, please see my comments, add those tests, then when they are all passing, it's good to be merged.

File: __tests__/app.test.js (hunk 1 above)
Comment (sunilkrishna49) [reply __tests__/app.test.js:344]: done

Comment (sunilkrishna49) [reply __tests__/app.test.js:344]: done

Comment (sunilkrishna49) [reply __tests__/app.test.js:344]: done

**Pull request merged**